src/extension.ts

```diff
@@ -113,19 +111,17 @@ function getServerOptions(context: vscode.ExtensionContext): languageclient.Serv
 
 function getClientOptions(): languageclient.LanguageClientOptions {
     return {
-        // Register the server for plain text documents
+        // Register the server for Azure Pipelines documents
         documentSelector: [
             { language: 'azure-pipelines', scheme: 'file' },
             { language: 'azure-pipelines', scheme: 'untitled' }
         ],
         synchronize: {
-            // Synchronize the setting section 'languageServerExample' to the server
-            // TODO: Are these what settings we want to pass through to the server? Would be good to see this happening... And see initializeOptions. Maybe remove them?
+            // TODO: Switch to handling the workspace/configuration request
             configurationSection: ['yaml', 'http.proxy', 'http.proxyStrictSSL'],
-            // Notify the server about file changes to '.clientrc files contain in the workspace
+            // Notify the server about file changes to YAML files in the workspace
             fileEvents: [
-                vscode.workspace.createFileSystemWatcher('**/*.?(e)y?(a)ml'),
-                vscode.workspace.createFileSystemWatcher('**/*.json')
```

Comment:
I can add this back since JSON is a subset of YAML but my understanding is that this is a _yaml_ language server for Azure Pipelines, which expects YAML files...